packages/openapi-generator/src/generator.ts

```diff
@@ -99,6 +105,14 @@ async function generateSDKSources(
     );
     await transpileDirectory(serviceDir);
   }
+
+  if (options.additionalFiles) {
```

Comment:
I see new `CHANGELOG.md` files but I do not see an option, are they copied via the additional files I guess? Perhaps you can adjust the PR title.